<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-T4B78EE" ignored="0" name="TestChrome" tests="3" failures="1" timestamp="20 Aug 2017 06:55:21 GMT" time="101.602" errors="0">
  <testcase name="testValidLogin" time="16.743" classname="com.orbitor.test.ValidLogin"/>
  <testcase name="testInvalidLogin" time="31.95" classname="com.orbitor.test.InvalidLogin"/>
  <testcase name="testInvalidLogin" time="13.509" classname="com.orbitor.test.VerifyProductVersion">
    <failure type="java.lang.AssertionError" message="expected [actiTIME 2017.1] but found [actiTIME 2017.3]">
      <![CDATA[java.lang.AssertionError: expected [actiTIME 2017.1] but found [actiTIME 2017.3]
	at com.orbitor.page.EnterTimeTrackPage.verifyVersion(EnterTimeTrackPage.java:41)
	at com.orbitor.test.VerifyProductVersion.testInvalidLogin(VerifyProductVersion.java:43)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testInvalidLogin -->
</testsuite> <!-- TestChrome -->
